### PR TITLE
Add explicit rule against spamming and all-caps

### PR DESCRIPTION
**Pull request by ClobberXD**
`No spamming, using all-caps, or any other method of chat abuse.`

If you feel that the wording can be improved, please advise.

### CODE REVIEW

**Comment by ClobberXD** [comment]
@rubenwardy **BUMP**

**Comment by ClobberXD** [comment]
I just rebased this PR. The horror...

**Comment by ClobberXD** [comment]
Rebased a one-line PR for the second time... -_-

File: init.lua [hunk 1]
```diff
@@ -4,30 +4,30 @@
...
 
--- Kpenguin, Thomas-S, Dragonop, stormchaser3000, Calinou, sparky/ircSparky.
 
 local items = {
 	"By playing on this server you agree to these rules:",
 	"",
-	"1. No swearing, dating, or other inappriopriate behaviour..",
-	"2. Don't be a cheater. No hacked clients.",
-	"3. Don't excessively spawnkill. Some spawn killing to stop players chasing",
+	"1. No swearing, dating, or other inappriopriate behaviour...",
```
Comment (rubenwardy) [inline init.lua]: this was meant to be a single dot

Comment (ClobberXD) [reply init.lua]: Oh lol. Fixed.

**Pull request merged**